tools/c7n_terraform/tests_terraform/test_parser.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 Cloud Custodian Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from c7n_terraform.parser import (
+    HclLocator, TerraformVisitor, Parser)
+from .tf_common import data_dir
+
+
+def test_parser_eof():
+    data = Parser().parse_module(data_dir / "aws-s3-bucket")
+    path = data_dir / "aws-s3-bucket" / "s3.tf"
+    assert path in data
+    assert len(data) == 3
+    tf_assets = data[path]
+    assert list(tf_assets) == ["resource"]
+    assert list(tf_assets["resource"][0]) == ["aws_s3_bucket"]
```

Comment:
What is the [0] for? That isn't clear.

Reply:
thats just the way all terraform assets are referenced, its more a standard artifact of hcl